app/helpers/locations_helper.rb

```diff
@@ -11,7 +11,7 @@ def country_link(country, count = nil)
   # title of a link to Observations at a location, with observation count
   # Observations at this Location(nn)
   def show_obs_link_title_with_count(location)
-    "#{:show_location_observations.t} (#{location.observations.count})"
+    "#{:show_location_observations.t} (#{location.observations.length})"
```

Comment:
Should this be `count` (or `size`)?
In a simple manual test with a breakpoint @ line 14, I get
`  Observation Load (559.3ms)  SELECT `observations`.* FROM `observations` WHERE `observations`.`location_id` = 2437
363
`

`(ruby) location.observations.count
   (192.0ms)  SELECT COUNT(*) FROM `observations` WHERE `observations`.`location_id` = 2437
`

`(ruby) location.observations.size
   (200.0ms)  SELECT COUNT(*) FROM `observations` WHERE `observations`.`location_id` = 2437
363
`

So to me it looks like `count` and `size` are 2x faster than `length`.

Reply:
Yes. Thank you.

Reply:
```suggestion
    "#{:show_location_observations.t} (#{location.observations.size})"
```
Putting a suggestion here so i can commit it right here.